### ISSUE TITLE
Blue Power RS Latch now goes crazy on every log in

### ISSUE THREAD

**Issue by pyure** [JasonMcRay/InfiTech-1.7#321]
When I log in, the RS latch is flipping several times per second between its two states.

Prior to updated to most recent version, it only did this once.  Now it does it every time I log on.

I suspect its hard to replicate: it will be a timing issue with all the redstone signals feeding into it when the world loads.  But the timing is screwy now, and my reactor exploded when it failed to receive correct signals (not an issue, restored backup)

BP is seriously starting to annoy me though.  I'm considering ripping it out and completely moving over to more-stable project red and to hell with the missing volcanos.

**Comment by JasonMcRay** [comment]
Could you please report it directly to BP tracker? Thanks :p

**Comment by pyure** [comment]
Jason, after fiddling with it for a while trying to set up a repeat-test case, I stopped being able to replicate the flashing behaviour. 

Then I ran into a probably-connected issue where the RS Latch was "forgetting" its state on world-reload, but I believe the culprit was GT: There's probably a bit of a delay with the GT5u Wireless cover in sending its signal, so the RS Latch thought there was no signal coming through and changed state.

I placed a repeater after the GT signal, so now when it comes through ( a tick later) the RS Latch re-updates to its proper state.

There's nothing I can really justifiably report to BP anymore.
